README.md

```diff
@@ -817,14 +816,17 @@ class User < ApplicationRecord
   ...
   before_save :downcase_unconfirmed_email
   ...
-  validates :unconfirmed_email, format: {with: VALID_EMAIL_REGEX, allow_blank: true}
-  validate :unconfirmed_email_must_be_available
+  validates :unconfirmed_email, format: {with: URI::MailTo::EMAIL_REGEXP, allow_blank: true}
 
   def confirm!
-    if unconfirmed_email.present?
-      update(email: unconfirmed_email, unconfirmed_email: nil)
+    if unconfirmed_or_reconfirming?
```

Comment:
This is an extra safeguard since we call this on the confirmations controller too.